README.md

```diff
@@ -28,7 +29,6 @@ Reading from digital zinc gage
 
 Output
 ---------
-A signal containing the output reading of the gage in bytes. 
+A signal containing the output reading of the gage (string) in mils.
```

Comment:
Put an example signal on here like this block https://raw.githubusercontent.com/nio-blocks/gpio/master/README.md